### PR TITLE
Add Solr Replication documentation

### PR DESCRIPTION
**Pull request by dspe**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (N/A)
| Versions      | 2.x 

Small documentation regarding set up a Solr Replication (Master / Slave). Tested on version 6.6.5 LTS

### CODE REVIEW

File: docs/guide/solr.md [hunk 1]
```diff
@@ -463,3 +463,86 @@ my_webinar_app.webinar_event_title_fulltext_field_mapper:
...
+
+And restart the master via
+
+```bash
+sudo su - solr -c "/opt/solr/bin/solr restart"
+```
+
+### Configuring Slave for replication
+
+You have to edit the same file on the slave server, and put the code bellow:
```
Comment (glye) [inline docs/guide/solr.md]:
```suggestion
You have to edit the same file on the slave server, and put the code below:
```
;)

File: docs/guide/solr.md [hunk 2]
```diff
@@ -463,3 +463,86 @@ my_webinar_app.webinar_event_title_fulltext_field_mapper:
...
+
+!!! note
+
+``` 
+The configuration bellow has been tested on Solr 6.5.5. Version 4.x should works also.
+```
+
+### Configuring Master for replication
+
+First we need to change the core configuration in *solrconfig.xml*  ( By example */opt/solr/server/ez/collection1/conf/solrconfig.xml ). You could copy and paste the code bellow before any other **requestHandler** section.
```
Comment (glye) [inline docs/guide/solr.md]:
```suggestion
First we need to change the core configuration in *solrconfig.xml*  ( By example */opt/solr/server/ez/collection1/conf/solrconfig.xml ). You could copy and paste the code below before any other **requestHandler** section.
```

File: docs/guide/solr.md [hunk 3]
```diff
@@ -46,9 +46,9 @@ bin/solr start -f -a "-Dsolr.solr.home=multicore"
 
 ##### Solr 6
 
-Download and extract Solr. Solr Bundle 1.3 and higher supports Solr 6 *(currently tested with Solr 6.6.0)*:
+Download and extract Solr. Solr Bundle 1.3 and higher supports Solr 6 *(currently tested & recommended with Solr 6.6LTS)*:
```
Comment (DominikaK) [inline docs/guide/solr.md]:
```suggestion
Download and extract Solr. Solr Bundle 1.3 and higher supports Solr 6 *(currently tested and recommended with Solr 6.6LTS)*:
```

File: docs/guide/solr.md [hunk 4]
```diff
@@ -463,3 +463,86 @@ my_webinar_app.webinar_event_title_fulltext_field_mapper:
     tags:
         - {name: ezpublish.search.solr.field_mapper.content}
 ```
+
+## Configuring Solr Replication (master/slave)
+
+!!! note
+
+```
```
Comment (DominikaK) [inline docs/guide/solr.md]:
```suggestion
```

File: docs/guide/solr.md [hunk 5]
```diff
@@ -463,3 +463,86 @@ my_webinar_app.webinar_event_title_fulltext_field_mapper:
     tags:
         - {name: ezpublish.search.solr.field_mapper.content}
 ```
+
+## Configuring Solr Replication (master/slave)
+
+!!! note
+
+``` 
+The configuration bellow has been tested on Solr 6.5.5. Version 4.x should works also.
```
Comment (DominikaK) [inline docs/guide/solr.md]:
```suggestion
    The configuration bellow has been tested on Solr 6.5.5. Version 4.x should works also.
```

File: docs/guide/solr.md [hunk 6]
```diff
@@ -463,3 +463,86 @@ my_webinar_app.webinar_event_title_fulltext_field_mapper:
...
+```
```
Comment (DominikaK) [inline docs/guide/solr.md]:
```suggestion
```

File: docs/guide/solr.md [hunk 7]
```diff
@@ -463,3 +463,86 @@ my_webinar_app.webinar_event_title_fulltext_field_mapper:
...
+
+### Configuring Master for replication
+
+First we need to change the core configuration in *solrconfig.xml*  ( By example */opt/solr/server/ez/collection1/conf/solrconfig.xml ). You could copy and paste the code below before any other **requestHandler** section.
```
Comment (DominikaK) [inline docs/guide/solr.md]:
```suggestion
First you need to change the core configuration in `solrconfig.xml` (for example */opt/solr/server/ez/collection1/conf/solrconfig.xml ).
You can copy and paste the code below before any other `requestHandler` section.
```

File: docs/guide/solr.md [hunk 8]
```diff
@@ -463,3 +463,86 @@ my_webinar_app.webinar_event_title_fulltext_field_mapper:
...
+  </lst>
+  <int name="maxNumberOfBackups">2</int>
+  <lst name="invariants">
+    <str name="maxWriteMBPerSec">16</str>
+  </lst>
+</requestHandler>
+<str name="confFiles">solrconfig_slave.xml:solrconfig.xml,x.xml,y.xml</str>
+```
+
+And restart the master via
```
Comment (DominikaK) [inline docs/guide/solr.md]:
```suggestion
Then restart the master with:
```

File: docs/guide/solr.md [hunk 9]
```diff
@@ -463,3 +463,86 @@ my_webinar_app.webinar_event_title_fulltext_field_mapper:
...
+
+```bash
+sudo su - solr -c "/opt/solr/bin/solr restart"
+```
+
+### Configuring Slave for replication
+
+You have to edit the same file on the slave server, and put the code below:
```
Comment (DominikaK) [inline docs/guide/solr.md]:
```suggestion
You have to edit the same file on the slave server, and use the code below:
```

File: docs/guide/solr.md [hunk 10]
```diff
@@ -463,3 +463,86 @@ my_webinar_app.webinar_event_title_fulltext_field_mapper:
...
+
+    <!-- If HTTP Basic authentication is enabled on the master, then the slave
+         can be configured with the following -->
+    <str name="httpBasicAuthUser">username</str>
+    <str name="httpBasicAuthPassword">password</str>
+  </lst>
+</requestHandler>
+```
+
+And restart Solr slave.
```
Comment (DominikaK) [inline docs/guide/solr.md]:
```suggestion
Next, restart Solr slave.
```

**Comment by andrerom** [comment]
@dspe be aware you change permission level on the solr.md file ;)

File: docs/guide/solr.md [hunk 11]
```diff
@@ -463,3 +463,85 @@ my_webinar_app.webinar_event_title_fulltext_field_mapper:
     tags:
         - {name: ezpublish.search.solr.field_mapper.content}
 ```
+
+## Configuring Solr Replication (master/slave)
+
+!!! note
+
+    The configuration bellow has been tested on Solr 6.5.5. Version 4.x should works also.
```
Comment (DominikaK) [inline docs/guide/solr.md]:
```suggestion
    The configuration below has been tested on Solr 6.5.5. Version 4.x should also work.
```

File: docs/guide/solr.md [hunk 12]
```diff
@@ -463,3 +463,85 @@ my_webinar_app.webinar_event_title_fulltext_field_mapper:
...
+
+### Configuring Master for replication
+
+First you need to change the core configuration in `solrconfig.xml` (for example */opt/solr/server/ez/collection1/conf/solrconfig.xml ).
```
Comment (DominikaK) [inline docs/guide/solr.md]:
```suggestion
First you need to change the core configuration in `solrconfig.xml` (for example `*/opt/solr/server/ez/collection1/conf/solrconfig.xml`).
```

**Comment by DominikaK** [comment]
@dspe, do you need anything from our side to move this PR forward?

**Comment by dspe** [comment]
@DominikaK i ùhave to check if everything is correct. I don't remember

File: docs/guide/solr.md [hunk 13]
```diff
@@ -463,3 +462,85 @@ my_webinar_app.webinar_event_title_fulltext_field_mapper:
...
+    <str name="httpBasicAuthPassword">password</str>
+  </lst>
+</requestHandler>
+```
+
+Next, restart Solr slave.
+
+Connect to the Solr slave interface (http://localhost:8983/solr), go to your core and check the replication status:
+
+![Solr Slave](C:\projects\developer-documentation\docs\guide\img\solr.PNG)
```
Comment (DominikaK) [inline docs/guide/solr.md]:
```suggestion
![Solr Slave](img/solr.PNG)
```

**Comment by DominikaK** [comment]
@glye @bdunogier or @andrerom, can you take a look at this again?

**Pull request merged**